api-model.gemspec

```diff
@@ -10,11 +10,13 @@ Gem::Specification.new do |s|
   s.summary     = "A simple way of interacting with rest APIs"
   s.description = "API model is a simple wrapper for interacting with external APIs. It tries to make it very simple and easy to make API calls and map the responses into objects."
 
-  s.add_dependency 'activesupport', '~> 4.1'
-  s.add_dependency 'activemodel', '~> 4.1'
-  s.add_dependency 'typhoeus', '~> 0.6'
+  # s.add_dependency 'activesupport', '~> 4.1'
+  # s.add_dependency 'activemodel', '~> 4.1'
+  s.add_dependency 'activesupport', '>= 5.0'
```

Comment:
I think this should be `>= 4.1`, just in case we've forgotten about something which is using this currently, and since none of the changes aren't backwards compatible.